### PR TITLE
Exception on server if an empty file input was inserted with 'appendChild' (fix #395)

### LINKED ISSUE #395: Exception on server if an empty file input was inserted with 'appendChild'
When a file input is inserted with `appendChild`, it [calls](https://github.com/DevExpress/testcafe-hammerhead/blob/e65bec4bb58a23006d3328a749301531b3f1d90a/src/client/sandbox/node/element.js#L234) `_onElementAdded` and it [calls](https://github.com/DevExpress/testcafe-hammerhead/blob/e65bec4bb58a23006d3328a749301531b3f1d90a/src/client/sandbox/node/element.js#L357) `addFileInputInfo`. Latter [calls](https://github.com/DevExpress/testcafe-hammerhead/blob/e65bec4bb58a23006d3328a749301531b3f1d90a/src/client/sandbox/node/element.js#L386) `infoManager.getFiles` for the input, and `getFiles` [returns](https://github.com/DevExpress/testcafe-hammerhead/blob/a1020bb8c4267a290d8f4564e8f72c9b488269b0/src/client/sandbox/upload/info-manager.js#L165) an empty `FileListWrapper`.

Empty FileListWrapper turns into empty `files` array on server side [here](https://github.com/DevExpress/testcafe-hammerhead/blob/8cdb7417a2a867d15263eb8be1f0a488277a0534/src/upload/form-data-entry.js#L35), that causes an exception `TypeError: Cannot read property 'name' of undefined` on a next statement.

### PR DESCRIPTION
**Pull request by LavrovArtem**
\cc @AndreyBelym @churkin 
@VasilyStrelyaev please check test name

### CODE REVIEW

File: test/server/upload-test.js [hunk 1]
```diff
@@ -372,4 +372,12 @@ describe('Upload', function () {
 
         expect(actual.toString()).eql(expected.toString());
     });
+
+    it('Should remove file info only', function () {
```
Comment (VasilyStrelyaev) [inline test/server/upload-test.js]: I don't understand the name. Probably, needs some details

Comment (VasilyStrelyaev) [inline test/server/upload-test.js]: As Artem proposed,
```
    it('Should remove only the information about hidden input', function () {
```

Comment (VasilyStrelyaev) [inline test/server/upload-test.js]: Can't we really mention the *an empty file input was inserted* condition here?

Comment (LavrovArtem) [inline test/server/upload-test.js]: I think that there is nothing to change, everything is all right

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by LavrovArtem** [comment]
FPR

**Comment by churkin** [comment]
lgtm

**Comment by AndreyBelym** [comment]
lgtm, cc \@inikulin

**Comment by inikulin** [comment]
lgtm

**Pull request merged**